src/docker/engine.rs

```diff
@@ -143,6 +146,69 @@ impl Engine {
     }
 }
 
+fn is_rootless(kind: EngineType) -> Option<bool> {
+    env::var("CROSS_ROOTLESS_CONTAINER_ENGINE")
+        .ok()
+        .and_then(|s| match s.as_ref() {
+            "auto" => None,
+            b => Some(bool_from_envvar(b)),
+        })
+        .or_else(|| (!kind.is_docker()).then_some(true))
+}
+
+#[must_use]
+fn is_docker_rootless(ce: &Path, msg_info: &mut MessageInfo) -> bool {
+    let mut cmd = Command::new(ce);
+    cmd.args(["info", "-f", "{{.SecurityOptions}}"])
+        .run_and_get_output(msg_info)
+        .ok()
+        .and_then(|cmd| cmd.stdout().ok())
+        .map(|out| {
+            out.to_lowercase()
+                .replace(['[', ' ', ']'], ",")
+                .contains(",name=rootless,")
```

Comment:
could this just be

```suggestion
            out.to_lowercase()
                .contains("name=rootless")
```

Reply:
It could, I just wanted to be safe.

The patch you suggest allows for all keys ending in `name` and all values starting with `rootless` :)

Reply:
Hi @Emilgardis All good with you?

Reply:
Sorry! Yes, this looks good